src/data/staking-products.json

```diff
@@ -546,9 +546,9 @@
       ],
       "isFoss": true,
       "hasBugBounty": false,
-      "isTrustless": false,
+      "isTrustless": true,
       "hasPermissionlessNodes": false,
-      "pctMajorityClient": null,
+      "pctMajorityClient": 43.1,
```

Comment:
Meta comment (unrelated to this PR) @wackerow can we keep this with even though we don't have a super majority? 

In the future would it need to evolve to cover > 1/3 clients?

Reply:
This is already a flag only for "majority" not "supermajority" implying the client would only need to have >50% of the network (still a risk).

We can think about a better way to rank these now that client diversity is improving. We may want to consider simply hiding this indicator if the network stats reach a certain quality threshold, where emphasis on client diversity isn't as critical. The goal here was to try and flag projects that were more strongly contributing to the supermajority problem, but this is already less critical.

Reply:
Perhaps it's worth opening a discussion on this as well, similar to #6452 for the "Trustless" tag